website/versioned_docs/version-v2.0_alpha/core-concepts/key-change.md

```diff
@@ -5,9 +5,9 @@ sidebar_label: Key change
 sidebar_position: 8
```

Comment:
please do not change v2.0 documentation, we should be creating a new folder for v3 documentation